estate/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import (estate_property, estate_property_offer, estate_property_tag,
+               estate_property_type)
```

Comment:
we tend to use this structure in the imports
```suggestion
from . import estate_property
from . import estate_property_offer
from . import estate_property_tag
from . import estate_property_type
```